gwtp-core/gwtp-mvp-client/src/main/java/com/gwtplatform/mvp/client/PresenterWidget.java

```diff
@@ -27,6 +27,7 @@
 import com.google.gwt.event.shared.GwtEvent;
 import com.google.gwt.event.shared.GwtEvent.Type;
 import com.google.gwt.event.shared.HasHandlers;
+import com.google.gwt.event.shared.UmbrellaException;
```

Comment:
If it's only for javadoc, use the FQN in the doc and add the simple name as the link text (`{@link com.google.gwt.event.shared.UmbrellaException UmbrellaException}`)